interface.go

```diff
@@ -65,6 +65,7 @@ type ContainerConfig struct {
 	HvRuntime                *HvRuntime  `json:",omitempty"` // Hyper-V container settings. Used by Hyper-V containers only. Format ImagePath=%root%\BaseLayerID\UtilityVM
 	Servicing                bool        // True if this container is for servicing
 	AllowUnqualifiedDNSQuery bool        // True to allow unqualified DNS name resolution
+	DNSSearchList            string      // List of DNS suffixes to use for resolution
```

Comment:
It probably would have been a better option. However,  I have already done the platform work for this and have it as a comma separated string there :(. I also don't expect this to change.